main/src/main/java/cgeo/geocaching/utils/ActionBarUtils.java

```diff
@@ -73,23 +83,57 @@ private static void setStableLayout(@NonNull final AbstractNavigationBarActivity
         }
     }
 
-    public static void setSystemBarAppearance(@NonNull final Activity activity, final boolean isActionBarShown) {
+    public static void setSystemBarAppearance(@NonNull final Activity activity, @Nullable final Boolean isActionBarShown) {
         final Window currentWindow = activity.getWindow();
         final WindowInsetsControllerCompat windowInsetsController = WindowCompat.getInsetsController(currentWindow, currentWindow.getDecorView());
 
         // set light/dark system bars depending on action bar colors
         final boolean isLightSkin = Settings.isLightSkin(activity);
         if (isLightSkin) {
-            final int actionBarColor = activity.getResources().getColor(R.color.colorBackgroundActionBar);
-            final boolean isLightStatusBar = !isActionBarShown || !ColorUtils.isBrightnessDark(actionBarColor);
-            windowInsetsController.setAppearanceLightStatusBars(isLightStatusBar);
+            final boolean supportsDarkStatusBars = (Build.VERSION.SDK_INT > Build.VERSION_CODES.P); // API 28
```

Comment:
Current workaround for UnifiedMapActivity, don't know, why it does not work for older versions. Have to check, which version is relevant.

Or any idea here?